File: components_page/components/spinner/button.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

spinners_button <- htmlDiv(
  list(
    dbcButton(
        dbcSpinner(size = "sm"),
        color = "primary",
        disabled = TRUE,
        className = "mr-1"
    ),
    dbcButton(
      list(dbcSpinner(size = "sm"), " Loading..."),
      color = "primary",
      disabled = TRUE
    )
  )
)

File: components_page/components/spinner/button.jl
using DashBootstrapComponents

spinners = html_div([
    dbc_button(
        dbc_spinner(size = "sm"),
        color = "primary",
        disabled = true,
        className = "me-1",
    ),
    dbc_button(
        [dbc_spinner(size = "sm"), " Loading..."],
        color = "primary",
        disabled = true,
    ),
]);
